image/Dockerfile.engine-dm

```diff
@@ -70,11 +70,27 @@ WORKDIR /tini
 RUN . /binaries-commits && \
     git checkout -q "$TINI_COMMIT" && \
     cmake . && make tini-static && \
-    cp tini-static /sbin/docker-init
+    cp tini-static /docker-init
 
 # Final docker image
 FROM scratch
-COPY --from=dockerd-builder /sbin/dockerd /sbin/
-COPY --from=proxy-builder /sbin/docker-proxy /sbin/
-COPY --from=init-builder /sbin/docker-init /sbin/
+ARG VERSION
+ARG GITCOMMIT
+ARG BUILDTIME
+ARG PLATFORM
+COPY --from=dockerd-builder /dockerd /sbin/
+COPY --from=proxy-builder /docker-proxy /sbin/
+COPY --from=init-builder /docker-init /sbin/
+
+LABEL \
+    org.opencontainers.image.authors="Docker Inc." \
+    org.opencontainers.image.created="${BUILDTIME}" \
+    org.opencontainers.image.documentation="https://docs.docker.com/" \
+    org.opencontainers.image.licenses="Apache-2.0" \
+    org.opencontainers.image.revision="${GITCOMMIT}" \
+    org.opencontainers.image.url="https://www.docker.com/products/docker-engine" \
+    org.opencontainers.image.vendor="Docker Inc." \
+    org.opencontainers.image.version="${VERSION}" \
+    distribution_based_engine="{\"platform\":\"${PLATFORM}\",\"containerd_min_version\":\"1.2.0-beta.1\",\"runtime\":\"host_install\"}"
```

Comment:
Same prefix "com.docker."

Reply:
Added in https://github.com/docker/docker-ce-packaging/pull/223/commits/4dae5046dac7f1467a611c20befd5cb2eb2aa7a3